Poor type safety in endCurrentTool caused err. 1096

bsb20 opened the pull request:
```
-This is not a permanent fix, but solves the problem while touching
at little as possible, and will not throw even if unexpected types
are received.
```

Commit:
Poor type safety in endCurrentTool caused err. 1096

    -This is not a permanent fix, but solves the problem while touching
    at little as possible, and will not throw even if unexpected types
    are received.

cwillisf commented on src/svgeditor/ImageEdit.as:
```diff
@@ -767,12 +768,14 @@ public class ImageEdit extends Sprite {
 			setToolMode("select");
 		}
 		// If the tool wasn't canceled and an object was created then select it
-		if (nextObject && nextObject.width > 0 && nextObject.height > 0 && (nextObject is Selection || nextObject.parent)) {
```
Your new code doesn't include `|| nextObject.parent`... do you know why that was in the old code?

Commit:
added check to endCurrentTool

    -Non-selection objects will be wrapped in a Selection
    -Selection items must have a parent, so we should check for this

cwillisf commented:
:+1:
